src/OrchardCore.Modules/OrchardCore.Setup/Startup.cs

```diff
@@ -33,7 +36,9 @@ public override void ConfigureServices(IServiceCollection services)
         {
             services.AddPortableObjectLocalization(options => options.ResourcesPath = "Localization");
             services.Replace(ServiceDescriptor.Singleton<ILocalizationFileLocationProvider, ModularPoFileLocationProvider>());
-
+            services.AddTransient<Microsoft.AspNetCore.Hosting.IStartupFilter, AutoSetupStartupFilter>();
```

Comment:
Maybe add an `using Microsoft.AspNetCore.Hosting`

Reply:
It will create conflicts with the namespaces of OrchardCore